app/src/main/res/values/strings.xml

```diff
@@ -2,6 +2,11 @@
     <string name="app_name">Apolline</string>
 
     <string name="todo">TODO</string>
+    <string name="ioio_regex">^ioio.</string>
```

Comment:
There is no point in doing this:
* Text for end users should be localised
* Internal encodings should be documented as constants